app/controllers/Healthcheck.scala

```diff
@@ -1,8 +1,10 @@
 package controllers
 
+import javax.inject.Inject
+
 import play.api.mvc._
 
-class Healthcheck extends Controller {
+class Healthcheck @Inject() extends Controller {
```

Comment:
I know!!!
To be fair, the other controller comes out looking a bit neater, although at the cost of it using impenetrable Dark Magiks.